packages/auth/karma.conf.js

```diff
@@ -54,10 +54,10 @@ function getTestFiles(argv) {
 }
 
 function getTestBrowsers(argv) {
-  let browsers = ["ChromeHeadless"]; 
+  let browsers = ['ChromeHeadless'];
```

Comment:
revert since it is not related to TOTP changes? Same for version.test.ts.

Reply:
I think this is a result of running yarn format. Auth was opted out for a long time. Christina got it back in scope but there may be a few spots that weren't caught

Reply:
For now, I have reverted these changes not related to this work